pyam/_aggregate.py

```diff
@@ -169,7 +179,7 @@ def _group_and_agg(df, by, method=np.sum):
     return df.groupby(cols).agg(_get_method_func(method))
 
 
-def _agg_weight(data, weight, method):
+def _agg_weight(data, weight, method, drop_negative):
```

Comment:
I used this recently and found the docstring a little confusing... it wasn't really intuitive that to perform a weighted average calculation, one should actually choose `method='sum'` - even if that makes sense from the formula...
I would consider adding a comment on this in the docstring

Reply:
Good idea!